.changeset/rotten-onions-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-astro': patch
+---
+
+correct wrong argument in test
```

Comment:
I don't think we need a changeset for internal test changes.

Reply:
@bluwy I reverted this change